### ISSUE TITLE
setup examples

### ISSUE THREAD

**Issue by aljosa** [pytest-dev/pytest-django#33]
i've tried to find examples on how to use new fixture mark w/ pytest_django.
something like:

```
import pytest
from django.contrib.sites.models import Site
from django.contrib.auth.models import User

@pytest.fixture(scope="session", autouse=True)
@pytest.mark.django_db
def project_setup(*args, **kwargs):
    user = User.objects.create(username="session user #1")
    assert User.objects.count(), 1
    site, created = Site.objects.get_or_create(id=1)
    site.domain = "testserver"
    site.save()
```

to do some db modifications after test db is ready.
this fails w/ error "Failed: Database access not allowed, use the "django_db" mark to enable"

could somebody provide an example on howto setup additional stuff after test db is ready?

**Comment by or** [comment]
This is quite old, but I had a similar problem and found a work-around that worked for me, so in case someone else stumbles across this:

```python
@pytest.fixture(autouse=True, scope='session')
def post_db_setup(_django_db_setup, _django_cursor_wrapper):
    with _django_cursor_wrapper:
        call_command('loaddata', *settings.STATIC_FIXTURES, interactive=False)
```

I realize fixtures are not the preferred way to do things, but if you *really* need to fill your DB with some values before any of the tests run... and only once, then that works quite well. It makes sure `_django_db_setup`  has been done and you get `_django_cursor_wrapper`  to write to it (I suspect any model creation would work in that context, but haven't tried it).

**Comment by mariocesar** [comment]
Have exactly the same problem, kind of frustrating. Will be great if this is documented, it's really a common pattern to populate a DB before running tests, and later cleanup files, attachments, logs, etc.  :)

The current updated solution for pytest==3.0.7 pytest-django==3.1.2
```
@pytest.fixture(scope='session')
def populate_nodes(request, django_db_setup, django_db_blocker):
    from project.app.models import Attachment
    call_command('populate', verbosity=0, interactive=False)

    # Cleanup
    def teardown():
        django_db_blocker.unblock()

        for attachment in Attachment.objects.all():
            path, ext = os.path.splitext(attachment.attach.path)
            for fpath in glob.glob(f'{path}*'):
                os.unlink(fpath)

    request.addfinalizer(teardown)
    request.addfinalizer(django_db_blocker.restore)
    return request
```

**Comment by pelme** [comment]
@mariocesar Did you see this example in the docs?

https://pytest-django.readthedocs.io/en/latest/database.html#populate-the-database-with-initial-test-data

I'm closing the issue for now (it was supposed to be closed when that example was added). Please reopen with additional details or open another issue if this needs to be further clarified or improved!
